html/script/app.js

```diff
@@ -1,138 +1,156 @@
-// INTRO ANIMATION
-
-areYouView();
+//CONST
+const today= new Date();
```

Comment:
beautiful <3 but one comment - you should start with SNAKE_CASE variables, then the camelCase, so position variables in order. ORDNUNG MUSS SEIN